models/quickstart.mdx

```diff
@@ -109,12 +109,27 @@ with wandb.init(project=project, config=config) as run:
         print(f"epoch={config['epochs']}, accuracy={acc}, loss={loss}")
         run.log({"accuracy": acc, "loss": loss})
 ```
-
-Visit [wandb.ai/home](https://wandb.ai/home) to view recorded metrics such as accuracy and loss and how they changed during each training step. The following image shows the loss and accuracy tracked from each run. Each run object appears in the **Runs** column with generated names.
+View recorded metrics such as accuracy and loss, and observe how they changed during each training step. You can view and monitor runs using the W&B App or the `wandb beta leet` terminal UI.
```

Comment:
Same here. I don't think we should add this feature to the landing page since it's in Beta.